src/components/prescription-expanded.component.tsx

```diff
@@ -3,7 +3,7 @@ import { PatientUuid } from "@openmrs/esm-framework";
 import { Tab, Tabs, TabList, TabPanels, TabPanel, Button } from "@carbon/react";
 import { useTranslation } from "react-i18next";
 import HistoryAndComments from "./history-and-comments.component";
-import styles from "./order-expanded.scss";
```

Comment:
I renamed "order-expanded" to "prescription-expanded" because it seems clearer with our current data model